Modules/Quotas/src/Quotas.Domain/Aggregates/Messages/Message.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace Backbone.Modules.Quotas.Domain.Aggregates.Messages;
 public class Message : ICreatedAt
 {
-    public string Id { get; set; }
-    public string CreatedBy { get; set; }
+    public string? Id { get; set; }
+    public string? CreatedBy { get; set; }
```

Comment:
I am not completely sure but I think this should not be nullable either.

Reply:
👍